integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/putmessage/StrategyFactory.java

```diff
@@ -38,10 +46,15 @@ public MessageStrategyFactory getFactory(ServiceRecord serviceRecord) {
             return eduMessageStrategyFactory;
         } else if ("POST_VIRKSOMHET".equalsIgnoreCase(serviceRecord.getServiceIdentifier())) {
             return postVirksomhetStrategyFactory;
+        } else if ("DPI".equalsIgnoreCase(serviceRecord.getServiceIdentifier())){
```

Comment:
Vi burde lage en enum for disse verdiene, ev. bruke noe som er definert i SR og dratt inn som avhengighet.